internal/core/src/common/Types.cpp

```diff
@@ -55,4 +55,24 @@ MetricTypeToName(MetricType metric_type) {
     return metric_bimap.right.at(metric_type);
 }
 
+bool
+IsPrimaryKeyDataType(DataType data_type) {
+    return data_type == engine::DataType::INT64 || data_type == DataType::VarChar;
```

Comment:
DataType::STRING cannot be primary key data type ?

Reply:
DataType::STRING is not supported now.